app/controllers/org_admin/templates_controller.rb

```diff
@@ -541,7 +536,7 @@ def calculate_table_scopes(templates, customizations)
       scopes = { all: templates.length, published: 0, unpublished: 0, dmptemplate_ids: templates.collect(&:dmptemplate_id).uniq }
       templates.each do |t|
         # If we have customizations use their status
-        if customizations.respond_to?(:keys)
+        if customizations.respond_to?(:keys) && customizations.keys.length > 0
```

Comment:
customization is defined as a hash above and does not get re-assigned so therefore .respond_to?(:keys) is not needed here. Can you remove it, please?

Reply:
good catch. will update